databases/week2/QestionNo.3.txt

```diff
@@ -0,0 +1,22 @@
+/* Get all the tasks assigned to users whose email ends in @spotify.com*/
+
+	select * from  task
+	inner join user on task.status_id=user.id
+	where user.email like '%@spotify.com';
+
+/* Get all the tasks for 'Donald Duck' with status 'Not started*/
+
+	select  *from  task
+	right join user on task.status_id=user.id
+	where user.name='Donald Duck' and task.status_id is null;
```

Comment:
The `task.status_id` column is not nullable, so the where clause doesn't really make sense. You should check which ID the status called "Not started" has